libs/net/dal/Migrations/1.0.25/Down/PostDown/00-SyndicationIngestUpdates.sql

```diff
@@ -0,0 +1,14 @@
+DO $$
+
+BEGIN
+
+DELETE FROM public.ingest_schedule
+WHERE ingest_id IN (
+    SELECT id FROM public.ingest WHERE TRIM(name) IN ('CBC | Aboriginal News', 'CBC | Top Stories', 'Link Newspaper')
```

Comment:
Why do you use `TRIM(name)`?

Reply:
In general we don't need it. However, I did see a case previously where data could be missed out due to an unintended error when editing data, so here just try making the query a bit more robust.